external/source/exploits/installerfiletakeover/InstallerFileTakeOver/ReflectiveLoader.c

```diff
@@ -0,0 +1,548 @@
+//===============================================================================================//

```

Comment:
This is already available at `external/source/ReflectiveDLLInjection/dll/src/ReflectiveLoader.c`.